apps/data/database/initdb.sql

```diff
@@ -32,7 +32,7 @@ CREATE TABLE listings (
     seller_id INTEGER NOT NULL REFERENCES users(user_id) ON DELETE CASCADE,
     buyer_id INTEGER REFERENCES users(user_id),
     title VARCHAR NOT NULL,
-    price INTEGER NOT NULL,
+    price REAL NOT NULL,
```

Comment:
We're currently using `DECIMAL(10,2)` in organizations to represent a money amount, this is much preferrable to a floating point format since floats are so lossy
```suggestion
    price DECIMAL(10,2) NOT NULL,
```
`DECIMAL(10,2)` is not perfect either (the ideal would probably be to store an exact bigint of cents) but we're not using real money so it's not a big deal if we're a cent or two off

Reply:
Sounds good to me, as long as the user is getting back what they're putting in I'm happy!